modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc

```diff
@@ -46,33 +39,32 @@ PrivateTmp=false
 ----
 ====
 
-[[configuring-your-web-server]]
-== Configuring Your Web server
+== Configuring Your Web Server
 
-NOTE: ownCloud comes with its own `owncloud/.htaccess` file. Because `php-fpm` can’t read
-PHP settings in `.htaccess` these settings must be set in the `owncloud/.user.ini` file.
+NOTE: ownCloud comes with its own `owncloud/.htaccess` file. 
+Because `php-fpm` can’t read PHP settings in `.htaccess` these settings must be set in the `owncloud/.user.ini` file.
 
-Set the following two parameters inside the corresponding php.ini file
-(see the *Loaded Configuration File* section of xref:configuration/general_topics/general_troubleshooting.adoc#php-version-and-information[PHP Version and Information]
-to find your relevant php.ini files) :
+Set the following two parameters inside the corresponding php.ini file (see the *Loaded Configuration File* section of xref:configuration/general_topics/general_troubleshooting.adoc#php-version-and-information[PHP Version and Information] to find your relevant php.ini files):
 
+[source,console]
 ----
 php_value upload_max_filesize = 16G
 php_value post_max_size = 16G
 ----
```

Comment:
Not needed anymore.

Reply:
That should be kept, but corrected to be `[source,php]` for proper syntax highlighting.